src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1750,6 +1763,33 @@ int xcldev::xclReset(int argc, char *argv[])
         return -EINVAL;
     }
 
+    std::string vbnv, errmsg;
+    auto dev = pcidev::get_dev(index);
+    dev->sysfs_get( "rom", "VBNV", errmsg, vbnv );
+    if (!errmsg.empty()) {
+        std::cout << errmsg << std::endl;
+        return -EINVAL;
+    }
+    if (!all && vbnv.find("_u30_") != std::string::npos) {
+        std::stringstream dbdf;
+        std::string output;
+        std::string xbresetPath = "/opt/xilinx/xrt/python/xbreset.py";
+        dbdf << std::setfill('0') << std::hex
+            << std::setw(4) << dev->domain << ":"
+            << std::setw(2) << dev->bus << ":"
+            << std::setw(2) << dev->dev << "."
+            << std::setw(1) << dev->func;
+        std::cout << "Card level reset. This will reset all FPGAs on the card." << std::endl;
+        int ret = isSudo();
+        if (ret)
+            return ret;
+        std::cout << "All existing processes will be killed." << std::endl;
```

Comment:
By the way I am afraid by the message content.
Is it safe to use XRT? :-)

Reply:
the msg was already part of the msg shown to user when a 'xbutil reset' is issued